panel/package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/holoviz/panel.git"
   },
   "dependencies": {
-    "@bokeh/bokehjs": "^2.4.3",
+    "@bokeh/bokehjs": "^3.0.0a11",
```

Comment:
I'm not sure how this resolves, I suppose this is some kind of alias, but the true version is `3.0.0-dev.11`.

Reply:
Good to know (I took it from the previous PR)